docs/book/src/user/quick-start.md

```diff
@@ -358,7 +361,21 @@ For more information about prerequisites, credentials management, or permissions
 {{#/tab }}
 {{#tab OpenStack}}
 
-Please visit the [OpenStack getting started guide].
+A ClusterAPI compatible image must be available in your OpenStack. For instructions on how to build a compatible image 
+see [image-builder (OpenStack)](https://image-builder.sigs.k8s.io/capi/providers/openstack.html). Depending on your OpenStack and underlying hypervisor it might also make sense to take a look at [image-builder (vSphere)](https://image-builder.sigs.k8s.io/capi/providers/vsphere.html).
+
+To see all required OpenStack environment variables execute:
+```bash
+clusterctl config cluster --infrastructure openstack:v0.3.0 --list-variables capi-quickstart
```

Comment:
You should be able to leave off the version here and have it select the right one from released versions.

Reply:
I just got a problem when doing it with aws. Something like could not discover default version. Do you know what caused this?

Reply:
Or if it works now I'll just retry and debug it if necessary. Just wasn't sure what to expect

Reply:
Ah, it's because the v0.3.0 release hasn't be cut yet :joy:

Reply:
Ah perfect then I'll release before merge here and then remove the version before the merge of this PR

Reply:
Ah there it is:
```
$ clusterctl init --infrastructure openstack
Error: failed to read "metadata.yaml" from the repository for provider "infrastructure-openstack": failed to download files from GitHub release v0.3.0-rc.0: failed to get file "metadata.yaml" from "v0.3.0-rc.0" release
```

Reply:
Fixed it directly. Problem was a missing file on the release or for a permanent solution a hardcoded contract struct for CAPO in clusterctl. Will open a PR for the struct as a follow up